### DISCUSSION TITLE
Sunbird Installation Issues on AWS

### DISCUSSION THREAD

**Discussion by Gowthamoptit** [Sunbird-Ed/Community#257] [Installation]
Hi team, 
I am trying to install sunbird on AWS and considering S3 service as a storage. But when I see the documentation I don't find any variables related to S3 storage bucket. 
Can you please suggest how we can use S3 bucket as a storage?

**Comment by beepdot** [comment]
Which release are you attempting to install @Gowthamoptit ?

**Reply by chitranshu-keshav** [reply]
@Gowthamoptit Please get back if the issue still persists else can you mark this thread as answered
